bbbeasy-backend/app/src/Core/Session.php

```diff
@@ -99,7 +100,18 @@ public function set($key, $value): void
         $this->f3->set('SESSION.' . $key, $value);
         $this->f3->sync('SESSION');
     }
+    public function getSession($sessionId)
```

Comment:
This is still problematic in production mode since session data is stored in redis in production environment.